### DISCUSSION TITLE
[taskbar-clock-customization] Font size on Windows 10

### DISCUSSION THREAD

**Discussion by ygoe** [ramensoftware/windhawk-mods#265] [General]
After setting up the clock customisation mod on Windows 10, my current main machine, I also tried it out in a virtual Windows 11 system to see how it works there. And it's much better than on Windows 10. Mostly because I can also change the font size and style. The default font is a little small and hard to read. I used T-Clock before this and also increased the font size a bit.

I understand that the Win11 taskbar uses a completely different rendering technology. But would it be possible anyway to change the font size of the clock also in Win10? If possible both lines separately and also the font weight, but the global size alone would probably help the most!

My Win11 settings are 18/13px (time, date), both at medium weight, the time with a character spacing of 25.

**Comment by m417z** [comment]
Right now, all the mod does for Windows 10 is changing texts, no UI-related stuff. Changing the clock visual style would be a separate research. I might look at it one day, but it's not a high priority so don't hold your breath.
